### PR TITLE
Fix issue mentioned in ticket #55

### PR DESCRIPTION
**Pull request by adjiap**
### SUMMARY
Implements necessary changes for #55 

*Reviewers:* @numere-org/maintainers

### IMPLEMENTATION
* Implementation: 
  - Added a simple `if (!event.GetItem().IsOk())` in both `DependencyDialog::OnItemActivate` and `DependencyDialog::OnItemActivate`
* Implementation test:
  - Opened one of the examples, /Procedures/tools/test.nscr
  - Opened Dependency Viewer box
  - Tried clicking, and double clicking on empty spaces.

### DOCUMENTATION
* [x] ChangesLog updated
* [ ] Code changes commented
* **Documentation articles:**
    * [ ] corresponding documentation articles updated
    * [ ] new documentation articles created
    * [x] not needed
* **Language files:**
    * [ ] corresponding language files updated
    * [x] not needed

----------------

### TESTS BY REVIEWERS
* [ ] Added to the automatic SW tests
* [x] Tested manually

### CODE REVIEW

File: Debug/NumeRe.res
Comment (numeredev) [inline Debug/NumeRe.res:1]: Remove this file from the branch

File: NumeRe.depend
Comment (numeredev) [inline NumeRe.depend:1]: Remove this file from the branch

File: NumeRe.layout
Comment (numeredev) [inline NumeRe.layout:1]: Remove this file from the branch

**Review by numeredev — CHANGES_REQUESTED** [review]
Remove the not necessary files please

**Comment by adjiap** [comment]
@numeredev Mistakenly commited files deleted in commit 31a1a699d01e1c89d59beed83ec6d3efff103063

**Review by numeredev — APPROVED** [review]

**Pull request merged**